### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by caoer** [musictheory/rchook#1]
Nice work! thanks for open source it, and all the notes. :+1: 

I am using [fastlane](https://github.com/fastlane/fastlane/) for creating builds. Have you compare the features with it?

**Comment by iccir** [comment]
Nope, looking at it now (thanks!).  Does fastlane gym (or one of the other tools) bump version numbers for you?

Having rchook always bump the build has saved us from MANY headaches.  I'm not the biggest fan of shell scripting and the actual rchook script itself (I'm sure somebody could make it more elegant!), but the mentality of "always bump to the next even version when you archive" is something I'd want if I ever adopted a different tool.

**Comment by caoer** [comment]
yes, it does. There are many [actions](https://github.com/fastlane/fastlane/blob/master/docs/Actions.md#modifying-project) can be used. include increasing build number. 

I think you tackle the same problem from different approaches. I have a fastlane script setup already, looks like don't have a reason to switch now.
